<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"><class>QPropCompare</class>
<widget class="QDialog" name="QPropCompare">
<property name="geometry">
<rect><x>0</x><y>0</y>
<width>510</width><height>440</height></rect></property>
<property name="windowTitle"><string>Compare</string></property>
<widget class="QGroupBox" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>14</y><width>482</width><height>100</height></rect></property>
<property name="title"><string>Whitespace</string></property>
<widget class="QRadioButton" name="IDC_WHITESPACE">
<property name="geometry">
<rect><x>14</x><y>24</y><width>412</width><height>20</height></rect></property>
<property name="text"><string>&amp;Compare</string></property>
</widget>
<widget class="QRadioButton" name="IDC_WHITE_CHANGE">
<property name="geometry">
<rect><x>14</x><y>46</y><width>412</width><height>20</height></rect></property>
<property name="text"><string>&amp;Ignore change</string></property>
</widget>
<widget class="QRadioButton" name="IDC_ALL_WHITE">
<property name="geometry">
<rect><x>14</x><y>68</y><width>412</width><height>20</height></rect></property>
<property name="text"><string>I&amp;gnore all</string></property>
</widget>
</widget>
<widget class="QGroupBox" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>328</y><width>482</width><height>64</height></rect></property>
<property name="title"><string>Comments</string></property>
<widget class="QCheckBox" name="IDC_FILTERCOMMENTS_CHECK">
<property name="geometry">
<rect><x>16</x><y>26</y><width>410</width><height>20</height></rect></property>
<property name="text"><string>Filter Comments</string></property>
</widget>
</widget>
<widget class="QCheckBox" name="IDC_IGNBLANKS_CHECK">
<property name="geometry">
<rect><x>14</x><y>120</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>Ignore blan&amp;k lines</string></property>
</widget>
<widget class="QCheckBox" name="IDC_IGNCASE_CHECK">
<property name="geometry">
<rect><x>14</x><y>144</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>Ignore &amp;case</string></property>
</widget>
<widget class="QCheckBox" name="IDC_EOL_SENSITIVE">
<property name="geometry">
<rect><x>14</x><y>168</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>Igno&amp;re carriage return differences (Windows/Unix/Mac)</string></property>
</widget>
<widget class="QCheckBox" name="IDC_CP_SENSITIVE">
<property name="geometry">
<rect><x>14</x><y>192</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>Ignore codepage &amp;differences</string></property>
</widget>
<widget class="QCheckBox" name="IDC_MOVED_BLOCKS">
<property name="geometry">
<rect><x>14</x><y>216</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>E&amp;nable moved block detection</string></property>
</widget>
<widget class="QCheckBox" name="IDC_MATCH_SIMILAR_LINES">
<property name="geometry">
<rect><x>14</x><y>240</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>&amp;Match similar lines</string></property>
</widget>
<widget class="QLabel" name="IDC_STATIC">
<property name="geometry">
<rect><x>14</x><y>264</y><width>280</width><height>40</height></rect></property>
<property name="text"><string>Diff &amp;algorithm (Experimental):</string></property>
<property name="inputMethodHints"><set>Qt::ImhMultiLine</set></property><property name="wordWrap"><bool>true</bool></property></widget>
<widget class="QComboBox" name="IDC_DIFF_ALGORITHM">
<property name="geometry">
<rect><x>290</x><y>264</y><width>200</width><height>22</height></rect></property>
<property name="sizeAdjustPolicy"><enum>QComboBox::AdjustToContents</enum></property></widget>
<widget class="QCheckBox" name="IDC_INDENT_HEURISTIC">
<property name="geometry">
<rect><x>14</x><y>292</y><width>482</width><height>20</height></rect></property>
<property name="text"><string>Enable indent &amp;heuristic</string></property>
</widget>
<widget class="QPushButton" name="IDC_COMPARE_DEFAULTS">
<property name="geometry">
<rect><x>14</x><y>396</y><width>140</width><height>28</height></rect></property>
<property name="text"><string>Defaults</string></property>
</widget>
</widget><resources/><connections/></ui>
